recipe/bld.bat

```diff
@@ -0,0 +1,29 @@
+mkdir build
+cd build
+
+cmake %SRC_DIR% -G "Ninja" ^
+      -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+      -DCMAKE_BUILD_TYPE="Release" ^
+      -DBUILD_PYTHON_BINDINGS=ON ^
+      -DBUILD_CLI_EXECUTABLES=ON ^
+      -DBUILD_SHARED_LIBS=OFF ^
+      -DBUILD_TESTS=OFF ^
+      -DMATLAB_BINDINGS=OFF ^
+      -DDOWNLOAD_STB_IMAGE=OFF ^
+      -DDOWNLOAD_ENSMALLEN=OFF ^
+      -DPYTHON_EXECUTABLE=%PYTHON% ^
+      -DUSE_OPENMP=ON ^
+      -DCMAKE_CXX_COMPILER=cl.exe ^
+      -DCMAKE_C_COMPILER=cl.exe ^
+      -DBoost_INCLUDE_DIRS=%LIBRARY_PREFIX%/include ^
+      -DBoost_USE_STATIC_LIBS=ON
```

Comment:
Huh, weird, it doesn't look like that changed anything.  Maybe it's work running with `Boost_LIB_DIAGNOSTIC_DEFINITIONS=1` to see what kind of extra output we get?